template.go

```diff
@@ -14,20 +14,23 @@ var generatedTmpl = template.Must(template.New("generated").Parse(`
 package {{.PackageName}}
 
 import (
+    "database/sql/driver"
     "encoding/json"
     "fmt"
+
+    "github.com/pkg/errors"
```

Comment:
Yup

On Fri, Jul 29, 2016 at 3:46 PM, Benjamin Trapani <notifications@github.com>
wrote:

> In template.go
> <https://github.com/clyphub/jsonenums/pull/1#discussion_r72850200>:
>
> >      "encoding/json"
> >      "fmt"
> > +
> > +    "github.com/pkg/errors"
>
> Should we prefer fmt.Errorf in this case?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/clyphub/jsonenums/pull/1/files/67c61e436f4bf250b03091496d05965ae91c1a3c..8ec93328be10f3ec5e3c7ca8d4f63dc1a0381d3a#r72850200>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABo58l91-WgzATbVjd7mJ79Ko60NWZAkks5qaliqgaJpZM4JYgdf>
> .
>



-- 


James Garfield

clypd | Principal Software Engineer



Connect with us: Twitter <https://twitter.com/clypd> | LinkedIn
<https://www.linkedin.com/company/clypd-inc-> | Facebook
<https://www.facebook.com/clypd>